### PR TITLE
Add yaml specific documentation.

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
## Description

Add `yaml_doc.md`.  

Please double check the file location, I parked it in `eva_user_guide` but I'll be glad to relocate it.

### CODE REVIEW

**Review by kevindougherty-noaa — APPROVED** [review]
Looks good to me!

**Review by CoryMartin-NOAA — COMMENTED** [review]
I think it has to be added to somewhere else so that it 'shows up', but not sure exactly where off the top of my head.

**Comment by CoryMartin-NOAA** [comment]
@EdwardSafford-NOAA can you add it to the list in eva_user_guide/index.rst

File: doc/eva_user_guide/eva_yaml.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Eva and yaml files
```
Comment (CoryMartin-NOAA) [inline doc/eva_user_guide/eva_yaml.md]: YAML

File: doc/eva_user_guide/eva_yaml.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Eva uses YAML formatted files to direct data load, data transformation, and 
+plot generation.  YAML is a human-readable data-serialization language.  It is 
+written with a .yml or .yaml (preferred) file extension.  Much like python, 
+indentation is critical in yaml.
+
+Eva contains examples of working YAML input files at `eva/src/eva/tests/config`.  
+
+## YAML general notes
+In YAML the `#` character proceeds comments.
```
Comment (CoryMartin-NOAA) [inline doc/eva_user_guide/eva_yaml.md]: `Precede`

File: doc/eva_user_guide/eva_yaml.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Eva contains examples of working YAML input files at `eva/src/eva/tests/config`.  
+
+## YAML general notes
+In YAML the `#` character proceeds comments.  
+
+Keys are identified by a following colon `mykey:`, and are followed by their corresponding value (which may be a single item, 
+other keys, lists, etc.).  
+
+List members are denoted by a leading dash and space `- mylist`, with one list member per line.  Lists can also be 
+specified by square brackets with each entry separated by a coma `[item1, item2]`.
```
Comment (CoryMartin-NOAA) [inline doc/eva_user_guide/eva_yaml.md]: comma !

**Review by CoryMartin-NOAA — COMMENTED** [review]
Couple comments, otherwise looks good

File: doc/eva_user_guide/eva_yaml.md (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply doc/eva_user_guide/eva_yaml.md]: Done.

File: doc/eva_user_guide/eva_yaml.md (hunk 2 above)
Comment (EdwardSafford-NOAA) [reply doc/eva_user_guide/eva_yaml.md]: Indeed.

File: doc/eva_user_guide/eva_yaml.md (hunk 3 above)
Comment (EdwardSafford-NOAA) [reply doc/eva_user_guide/eva_yaml.md]: I think I'm a bit tired.  Good eye.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**